metrics/storage/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2018 Intel Corporation
```

Comment:
I wonder if this is the best place to put the Dockerfile, or maybe the best name to use.
Given we are likely to gain more storage tests, unless we intend to add support for all of those tests to this one Dockerfile, then I suspect we will want to move or rename this one before we merge.
Also, iirc, `docker build` incorporates all the files in the same directory as the dockerfile into the image? (https://docs.docker.com/engine/reference/commandline/build/#options), so this might not be what we want here...

Over on CC we had started to hold all the dockerfiles in a separate tree a little higher up in the dir hierarchy - I think it will actually be better to keep the dockerfiles much closer to the associated tests if we can (which I think will only become a problem if we start sharing images across tests across trees...). Thus, I suggest we make a `blogbench` subdir here and place the dockerfile in that.

What do you think @GabyCT @chavafg ?